porcupine/plugins/directory_tree.py

```diff
@@ -461,9 +461,7 @@ def setup() -> None:
             tree.add_project(path, refresh=False)
     tree.refresh()
 
-    # TODO: mac right click = button 2?
-    # TODO: invoking context menu from keyboard
```

Comment:
It would be good to leave the keyboard comment here. You currently need a mouse click to open the menu, even though the directory tree can otherwise be used with just the keyboard.